tools/coq_dune.ml

```diff
@@ -0,0 +1,210 @@
+(* ocamlfind ocamlopt -linkpkg -package str coq_dune.ml -o coq_dune.opt *)
+open Format
+
+(* Keeping this file self-contained as it is a "bootstrap" utility *)
+(* Is OCaml missing these basic functions in the stdlib?           *)
+module Aux = struct
```

Comment:
In the code below `let f x = match x with ... no use of x ...` should be replaced by `let f = function ...`

Reply:
I guess this is a matter of style and we don't enforce it in Coq, right?

Reply:
I do

Reply:
> I do

What does this mean?

Reply:
I gave you a reason that is not aestetic. And it has nothing to do with Coq.
You are free to think it is aestetic, but this is not what I think. I'm not bykeshedding your code.

Hence (I do) care about this matter and I'm asking to use `function` when the input is use linearly and immediately destructed. `function` plays this role in OCaml.